python/iot3/src/iot3/mobility/etsi.py

```diff
@@ -87,7 +87,8 @@ def si2etsi(
                       dict with keys "min" and "max", in ETSI scale; the bounds are
                       inclusive, but must not include undef and out_of_range.
         :param out_of_range: the special ETSI-scaled value to use when the value is
-                             out of range
+                             out of range; is out_of_range is None, the encoded
```

Comment:
@ymorin-orange This is a comment that should have been on the CPM PR review, I realized that this one was coming after when I hit the CPM introduction commit
Then I forgot the comment here when I switched ...

Reply:
Not a refresh problem imo, I previously started the review with this comment, I think Github kept it for me despite reloading the page (as it told me it was outdated)
Anyway I left the comment just in case the typo was still there